client/jetpack-cloud/sections/comparison/table/useComparisonData.tsx

```diff
@@ -228,14 +228,38 @@ export const useComparisonData = () => {
 						icon: StatsIcon,
 						info: {
 							FREE: {
-								content: translate( 'Basic Stats' ),
+								content: (
+									<>
+										{ translate( 'Basic stats' ) }
+										<br data-screen="desktop" />
+										{ /* Space between description and parenthasis on mobile */ }
```

Comment:
Minor typo in the comments for all 3 columns
```suggestion
										{ /* Space between description and parenthesis on mobile */ }
```

Reply:
Thanks for the catch!